IFC 4.3/Align_1/UT_Align_1.IFC

```diff
@@ -0,0 +1,103 @@
+#33=IFCRELAGGREGATES('2uInlzG_X1_glgN9RNs$c8',#9,$,$,#30,(#32));
+
+/* Alignment: horizontal and vertical alignment */
+#34=IFCALIGNMENT('0XTxY6KDrCdQRFzSOb6jwJ',$,'UT_Align_1',$,$,$,$,#35,$);
+#35=IFCALIGNMENTCURVE(#36,#73,$);
+
+/* Horizontal alignment */
+#36=IFCALIGNMENTHORIZONTAL('06__Wngff6zAUzQvB4Fnt4',$,$,$,$,$,$,0.,(#60,#38,#40,#42,#44,#46,#62,#48,#50,#52,#64,#54,#56,#58,#66));
+#37=IFCCARTESIANPOINT((335095.95046541,4084593.74863239));
+#38=IFCALIGNMENTHORIZONTALSEGMENT($,$,#37,357.802095555,-25.,-25.,39.84063746,$,.CIRCULARARC.);
+#39=IFCCARTESIANPOINT((335121.90623167,4084618.34196855));
+#40=IFCALIGNMENTHORIZONTALSEGMENT($,$,#39,89.110110726,-25.,0.,9.,$,.CLOTHOID.);
+#41=IFCCARTESIANPOINT((335120.96892797,4084627.28000352));
+#42=IFCALIGNMENTHORIZONTALSEGMENT($,$,#41,99.423351036,0.,22.,10.22727273,$,.CLOTHOID.);
+#43=IFCCARTESIANPOINT((335120.08215878,4084637.4441296));
+#44=IFCALIGNMENTHORIZONTALSEGMENT($,$,#43,86.105633688,22.,22.,45.65445608,$,.CIRCULARARC.);
+#45=IFCCARTESIANPOINT((335153.94723443,4084654.44351607));
+#46=IFCALIGNMENTHORIZONTALSEGMENT($,$,#45,327.20528601,22.,0.,18.18181818,$,.CLOTHOID.);
+#47=IFCCARTESIANPOINT((335201.01029344,4084587.89698736));
+#48=IFCALIGNMENTHORIZONTALSEGMENT($,$,#47,303.529344066,0.,-50.,40.5,$,.CLOTHOID.);
+#49=IFCCARTESIANPOINT((335227.52147844,4084557.67048984));
+#50=IFCALIGNMENTHORIZONTALSEGMENT($,$,#49,326.734134759,-50.,-50.,79.33785465,$,.CIRCULARARC.);
+#51=IFCCARTESIANPOINT((335297.1868326,4084572.7216982));
+#52=IFCALIGNMENTHORIZONTALSEGMENT($,$,#51,57.648619296,-50.,0.,32.,$,.CLOTHOID.);
+#53=IFCCARTESIANPOINT((335311.14815049,4084614.6579186));
+#54=IFCALIGNMENTHORIZONTALSEGMENT($,$,#53,75.983268735,0.,60.,41.66666667,$,.CLOTHOID.);
+#55=IFCCARTESIANPOINT((335325.75784196,4084653.44126307));
+#56=IFCALIGNMENTHORIZONTALSEGMENT($,$,#55,56.088900855,60.,60.,27.60658519,$,.CIRCULARARC.);
+#57=IFCCARTESIANPOINT((335345.80042437,4084672.07101789));
+#58=IFCALIGNMENTHORIZONTALSEGMENT($,$,#57,29.726553879,60.,0.,41.66666667,$,.CLOTHOID.);
+#59=IFCCARTESIANPOINT((335085.95782221,4084594.13214501));
+#60=IFCALIGNMENTHORIZONTALSEGMENT($,$,#59,357.802095555,0.,0.,10.,$,.LINE.);
+#61=IFCCARTESIANPOINT((335165.88241545,4084640.91041051));
+#62=IFCALIGNMENTHORIZONTALSEGMENT($,$,#61,303.529344066,0.,0.,63.59552536,$,.LINE.);
+#63=IFCCARTESIANPOINT((335308.14596689,4084602.63177967));
+#64=IFCALIGNMENTHORIZONTALSEGMENT($,$,#63,75.983268735,0.,0.,12.39520568,$,.LINE.);
+#65=IFCCARTESIANPOINT((335385.54643726,4084683.81177391));
+#66=IFCALIGNMENTHORIZONTALSEGMENT($,$,#65,9.832185999,0.,0.,35.3941232,$,.LINE.);
+
+/* Alignment in Site */
+#67=IFCRELCONTAINEDINSPATIALSTRUCTURE('3DvGg4YE9DwRm6tfQg$num',$,$,$,(#34),#30);
+
+/* Vertical alignment */
+#68=IFCALIGNMENTVERTICALSEGMENT($,$,0.,14.256,365.8,0.0784810126582277,0.0784810126582277,$,.CONSTANTGRADIENT.);
+#69=IFCALIGNMENTVERTICALSEGMENT($,$,14.256,129.488,366.918825316456,0.0784810126582277,0.0784810126582277,890.,.PARABOLICARC.);
```

Comment:
A vertical segment that isn't constant should provide distinct start and end gradients.  Please review and adjust.

Reply:
Fixed in 1954276